examples/python/wasm_rust_psi_payload/Makefile

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 all:
-	@cargo build --target wasm32-unknown-unknown --release
+	@RUSTFLAGS= cargo build --target wasm32-unknown-unknown --release
```

Comment:
Looks a bit weird, how about `@RUSTFLAGS='' cargo build...` if it works.

Reply:
Good suggestion. Updated.